src/main/java/uk/gov/digital/ho/hocs/workflow/domain/repositories/ScreenRepository.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.digital.ho.hocs.workflow.domain.repositories;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.workflow.domain.repositories.entity.Schema;
+
+@Service
+public class ScreenRepository extends JsonFileReader {
+
+    protected ScreenRepository(ObjectMapper objectMapper) {
+        super(objectMapper);
+    }
+
+    //TODO: investigate cachable benefits
```

Comment:
Having run this locally some of the files took around 0.02ms to load, with repeat loads of < 0.005ms. I'm not convinced caching is necessary for this at the moment.